lambda_layers/core_layer/python/core_layer/alembic/env.py

```diff
@@ -4,7 +4,7 @@
 from sqlalchemy import engine_from_config
 from sqlalchemy import pool
 
-from alembic import context
```

Comment:
This change needs to be reverted, because the pipeline does not recognize the import statement anymore
![image](https://user-images.githubusercontent.com/25669997/124236560-d7ba3400-db16-11eb-9df8-425c41892897.png)